gapis/replay/batch.go

```diff
@@ -269,7 +269,7 @@ func (m *manager) execute(
 	}
 	ctx = log.V{"replay target ABI": replayABI}.Bind(ctx)
 
-	connection, err := m.connect(ctx, d, replayABI)
+	conn, err := m.connect(ctx, d, replayABI)
```

Comment:
I would be keen to keep the long, explicit `connection` name.

Reply:
there is a struct with the same name. I think in this situation shortening is better to not get confused.

Reply:
Fair enough.